### PR TITLE
Syslog

### LINKED ISSUE #54: permit logging to file instead of stdout/stderr
It'd be nice to be able to direct output to `/usr/local/var/log/sillymud/XXXX` so we can look at things there instead of `nohup.out` which is how I'm running the prod instance currently. I suspect there is probably a good C logging library/best practice that could be used here.

### PR DESCRIPTION
**Pull request by dangitall**
Instead of shoving everything to stderr, uses `syslog` to log if it is running. Right now everything is logged with the `LOG_INFO` severity. We could potentially have different levels in the future and handle them differently.

I added `/etc/rsyslog.d/20-silly.conf` on the server to log all silly messages to `/usr/local/var/sillymud/silly.log`. I don' t know if you have some instance provisioning this should go into.

Fixes #54

### CODE REVIEW

File: src/utility.c [hunk 1]
```diff
@@ -28,6 +30,7 @@ void log_msgf(const char *fmt, ...) {
   char buf[256];
   va_list args;
   va_start(args, fmt);
+  syslog(LOG_INFO, fmt, args);
```
Comment (jonm) [inline src/utility.c]: should this be `vsyslog` instead of `syslog`?

Comment (dangitall) [inline src/utility.c]: Yes!!!

File: src/utility.c [hunk 2]
```diff
@@ -342,18 +345,9 @@ int strn_cmp(char *arg1, char *arg2, int n) {
 
 /* writes a string to the log */
 void log_sev(char *str, int sev) {
```
Comment (jonm) [inline src/utility.c]: Given that this no longer writes to `stderr`, and only to the logged-in wizards, should this get renamed now? (at least, it needs a new comment!)

Comment (dangitall) [inline src/utility.c]: :+1:

**Comment by dangitall** [comment]
Hmm, I didn't realize anyone was calling `log_sev` directly before. I guess those should be changed to logs with the appropriate severity. I'll take a look.

**Comment by jonm** [comment]
I don't have any instance provisioning set up yet; for now I think it is sufficient to document it in the `INSTALL` instructions and then I will take a snapshot of the current filesystem.

**Comment by jonm** [comment]
I am :+1: to merge after you finish your investigation of the direct `log_wiz` invocations.

**Pull request merged**